package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@storybook/addon-storysource": "^5.2.8",
     "classnames": "2.2.6",
-    "gatsby": "2.17.15",
+    "gatsby": "^2.18.6",
```

Comment:
@BolajiAyodeji is this fine?

Reply:
yeah when i did npm install that changed

Reply:
I don't think `npm install` should upgrade gatsby for you. Are you certain that this got upgraded when you ran `npm install`?